### DISCUSSION TITLE
Where is the development history and changelog?

### DISCUSSION THREAD

**Discussion by BsNoSi** [datenstrom/community#554] [Ask a question] [answered]
I am "Yellow part time customizer" using spare project time with variable time space between other projects I have to do. Therefore I regularly miss updates "in between" which leads regularly to very gruelling sessions upgrading existing sites.

For me it would be very helpful to have a "changes diary" of the versions: New, changed, deprecated, removed. Simply to have an idea why things that worked perfectly are nonfunctional or "show stopper" after an update.

It is very time consuming to install previous an new versions side by side comparing them file by file and go into deep to understand what may be the reason for failure. 

The requirements for such a document would be low. Just a few key points, what has changed where, perhaps with a short "why", would help people like me, who are not part of the constant development, to find out if time can be invested in Yellow again. Above all, there would be more time for new things instead of chasing after old ones.

A "latest first" list of changes would allow the quick identification of possible problems, because it would make the gap between "then" and "now" very quickly tangible.

**Comment by markseuffert** [comment]
[There are announcements](https://github.com/datenstrom/community/discussions/categories/write-an-announcement) and for everything else you can [ask questions](https://github.com/datenstrom/community/discussions/categories/ask-a-question).

We are a small team and generally aim to make the update as smooth as possible. If you would like to have a better update process, you should join a team that tests releases and gives timely feedback. If you would like to have a better documentation, you should join a team that collects and creates detailed documentation. So I am passing the ball back to you. Maybe there's something you can do to make Datenstrom Yellow better?

**Comment by BsNoSi** [comment]
I know from other projects, especially commercial products for which I have been very successful in marketing and sales for many years, that there is a defined place for a changelog with changes to the central system. Based on this, further documents are created and maintained. Among other things, there is a note that documents need to be revised or created - which others can then do in a targeted manner.

As far as I have time for this, I am happy to contribute to it. However, as already mentioned, I have no possibility to do this continuously. Therefore, with the current form of information flow, it is difficult for me to get to the point where I can contribute something new or additional.

My time currently available for Yellow is almost used up and the actual goal to be up to date with several websites and plugins developed for them, including updates of my [repositories](https://github.com/BsNoSi?tab=repositories), is in undefined distance. 

At the moment I see the danger that I will have to break this off and that the distance until the next "session" will be even greater. Then possibly with even greater frictional losses, which would ultimately leave me behind. Which would make any form of support on my part impossible.

In my opinion the "pick list with keyword informations" is missing, offering topics to be worked through *by others* in a targeted manner, which in the end benefits everyone. If most of the time has to be spent by people who want to work, trying to find out where something needs to be done or could be done, it ultimately slows everyone down.

The most successful "CMS" for change logs I know is a "changes.txt" as part of the package. In case of YELLOW, I'd prefer a "changes.md" that is part of the install sample data.

**Comment by markseuffert** [comment]
Thanks for your feedback. I personally don't plan to do more detailed release notes.

**Comment by BsNoSi** [comment]
Maybe there is a misunderstanding. 

I do not expect or want "more details". It's simply a question of "where to find all". If  a "changes.md" would be part of the download package there is – in my opinion – no need for further places. The advantage would be having the history automatically for those missing changes between their installation to the current.

Possibly this may less to do than currently because there is no need to change existing documents but only adding the lates information at the beginning of an existing file leaving the previous entries below.

**Reply by nzyeehaa** [reply]
March 2026 - link no longer works due to discussions and releases being separated
~~July-2025 - Update - fixed url link~~

~~@BsNoSi I realise this is an old thread, but this may help someone looking for the same information as you...~~

~~[discussions/has been released](https://github.com/datenstrom/community/discussions?discussions_q=has+been+released)~~

~~...this will show all the releases, then each release has its own update information.~~
If you start from the version above your current website version, you may be able to guess how that update will affect your current website. (of course the website updater will not simply take you to the next version above your current version, it will attempt to get you updated all the way to the latest version)

In my experience, any API changes may possibly break your site, and your template may need updating.

**Comment by markseuffert** [comment, marked as answer]
Now there's a [what's new on the official website](https://datenstrom.se/yellow/help/what-s-new). Here you'll find the  release notes, all in one place.
